USAGE.md

```diff
@@ -125,21 +125,21 @@ The signature is passed via the -signature flag.
 It can be a file:
 
 ```shell
-$ cosign upload -signature file.sig dlorenc/demo
+$ cosign attach signature -signature file.sig dlorenc/demo
```

Comment:
can drop `-signature` in this case and just have 2 positionals

Reply:
I just went back and forth on this one and decided against it. -signature is long but two positionals is uglier

Reply:
/shrug